bin/oref0-setup.sh

```diff
@@ -247,14 +247,22 @@ fi
 mkdir -p $HOME/src/
 if [ -d "$HOME/src/oref0/" ]; then
     echo "$HOME/src/oref0/ already exists; pulling latest"
-    (cd ~/src/oref0 && git fetch && git pull) || die "Couldn't pull latest oref0"
+    cd ~/src/oref0
+    OLD_HEAD=$(git rev-parse HEAD)
+    (git fetch && git pull) || die "Couldn't pull latest oref0"
+    NEW_HEAD=$(git rev-parse HEAD)
+    if [ "$OLD_HEAD" == "$NEW_HEAD" ];  then
```

Comment:
Perhaps we should add a condition here that `git branch` is not on master.  If it is, we should be able to just install the packaged version...